dev/values/rollup/dev.yaml

```
@@ -2,129 +2,262 @@ global:
+      # rate.
+      sequencerBlockTimeMs: 2000
+      # URL path for the sequencer
+      sequencerRpc: "http://node0-sequencer-rpc-service.astria-dev-cluster.svc.cluster.local:26657"
+      # gRPC path for the sequencer
+      sequencerGrpc: "http://node0-sequencer-grpc-service.astria-dev-cluster.svc.cluster.local:8080"
+      # The maximum number of requests to make to the sequencer per second
+      sequencerRequestsPerSecond: 500
+
+    celestia:
+      rpc: "http://celestia-service.astria-dev-cluster.svc.cluster.local:26658"
+      token: "http://celestia-service.astria-dev-cluster.svc.cluster.local:5353"
+
+  resources:
+    conductor:
+      requests:
+        cpu: 0.01
+        memory: 1Mi
+      limits:
+        cpu: 0.1
+        memory: 20Mi
+    geth:
+      requests:
+        cpu: 0.25
+        memory: 256Mi
+      limits:
+        cpu: 2
+        memory: 1Gi
+
+  celestia-node:
+    enabled: false
+
+  storage:
+    enabled: false
+
+  ingress:
     enabled: true
-    privateKey:
-      devContent: "8b3a7999072c9c9314c084044fe705db11714c6c4ed7cddb64da18ea270dd203"
+    services:
+      rpc:
+        enabled: true
+      ws:
+        enabled: true
 
-    # The amount of token to give per request
-    amount: 1
-  sequencer:
-    chainId: sequencer-test-chain-0
-    # URL path for the sequencer
-    rpc: "http://node0-sequencer-rpc-service.astria-dev-cluster.svc.cluster.local:26657"
-    # gRPC path for the sequencer
-    grpc: "http://node0-sequencer-grpc-service.astria-dev-cluster.svc.cluster.local:8080"
+composer:
+  enabled: true
+  config:
+    sequencerChainId: sequencer-test-chain-0
+    sequencerRpc: "http://node0-sequencer-rpc-service.astria-dev-cluster.svc.cluster.local:26657"
     privateKey:
       devContent: "2bd806c97f0e00af1a1fc3328fa763a9269723c8db8fac4f93af71db186d6e90"
-  celestia:
-    rpc: "http://celestia-service.astria-dev-cluster.svc.cluster.local:26658"
-    token: "http://celestia-service.astria-dev-cluster.svc.cluster.local:5353"
-    initialBlockHeight: "2"
-    heightVariance: "10"
-  blockscout:
+    rollups:
+      - name: "astria"
+        wsRpc: "ws://astria-evm-service.astria-dev-cluster.svc.cluster.local:8546"
+
+bridge-withdrawer:
+  enabled: true
+  config:
+    sequencerCometbftEndpoint: "http://node0-sequencer-rpc-service.astria-dev-cluster.svc.cluster.local:26657"
+    sequencerChainId: "sequencer-test-chain-0"
+    minExpectedFeeAssetBalance: "0"
+    sequencerBridgeAddress: "astria13ahqz4pjqfmynk9ylrqv4fwe4957x2p0h5782u"
+    feeAssetDenom: "nria"
+    rollupAssetDenom: "nria"
+    evmContractAddress: "0xA58639fB5458e65E4fA917FF951C390292C24A15"
+    evmRpcEndpoint: "ws://astria-evm-service.astria-dev-cluster.svc.cluster.local:8546/"
+    sequencerPrivateKey:
+      devContent: "dfa7108e38ab71f89f356c72afc38600d5758f11a8c337164713e4471411d2e0"
+
+evm-faucet:
+  enabled: true
+  ingress:
     enabled: true
+  config:
+    rollupName: flame
```

default name is `astria` not `flame`
```suggestion
    rollupName: astria
```